jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -343,6 +344,10 @@ def collapse_to_spectrum(self, add_data=True, **kwargs):
                 sender=self)
             self.hub.broadcast(snackbar_message)
 
+        # per https://jwst-docs.stsci.edu/jwst-near-infrared-camera/nircam-performance/nircam-absolute-flux-calibration-and-zeropoints # noqa
+            pix_scale_factor = self.aperture.scale_factor * spectral_cube.meta.get('PIXAR_SR', 1.0)
+            collapsed_spec.meta['_pixel_scale_factor'] = pix_scale_factor
```

Comment:
I'm torn whether this belong in or outside of the `if add_data`... but might be useful for comparing if its outside (and then just in case anyone passes that data back into jdaviz)
```suggestion
        # per https://jwst-docs.stsci.edu/jwst-near-infrared-camera/nircam-performance/nircam-absolute-flux-calibration-and-zeropoints # noqa
        pix_scale_factor = self.aperture.scale_factor * spectral_cube.meta.get('PIXAR_SR', 1.0)
        collapsed_spec.meta['_pixel_scale_factor'] = pix_scale_factor
```

Reply:
oh but this needs to be done before `add_results_from_plugin`, right now the metadata entry isn't showing up when accessing through `viz.get_data(...)`.